templates/base/head_navbar.tmpl

```diff
@@ -1,6 +1,6 @@
 <div class="ui container" id="navbar">
 	<div class="item brand" style="justify-content: space-between;">
-		<a href="{{AppSubUrl}}/">
+		<a href="{{AppSubUrl}}/" title="{{if .IsSigned}}{{.i18n.Tr "dashboard"}}{{else}}{{.i18n.Tr "home"}}{{end}}">
```

Comment:
```suggestion
		<a href="{{AppSubUrl}}/" class="item poping up" data-content="{{if .IsSigned}}{{.i18n.Tr "dashboard"}}{{else}}{{.i18n.Tr "home"}}{{end}}" data-variation="tiny inverted">
```

Reply:
I think we should stick to the same style

![Screen Shot 2021-09-19 at 23 22 53](https://user-images.githubusercontent.com/844963/133942025-dcdb4f8f-cb9f-4a04-b568-61763a8e278d.png)
![Screen Shot 2021-09-19 at 23 21 54](https://user-images.githubusercontent.com/844963/133942026-1ed3ca0a-5a71-4f32-beb1-0b58a1f106b5.png)